src/safeds/ml/classical/regression/_arima.py

```diff
@@ -70,9 +71,10 @@ def fit(self, time_series: TimeSeries) -> ArimaModelRegressor:
         """
         from statsmodels.tsa.arima.model import ARIMA
 
-        if not isinstance(time_series, TimeSeries) and isinstance(time_series, Table):
-            raise NonTimeSeriesError
-        if time_series.number_of_rows == 0:
+        if not isinstance(time_series, TimeSeriesDataset) and isinstance(time_series, Table):
+            raise NonTimeSeriesDatasetError
```

Comment:
Do you still need this, as the `TimeSeriesDataset` does no longer inherit the `Table`, so the value cannot be a `Table`

Reply:
this is correct, removed